frontend/components/SetupPage/FundYourAgent/hooks/useGetRefillRequirementsWithMonthlyGas.ts

```diff
@@ -14,8 +14,8 @@ import { Address } from '@/types/Address';
 import { bigintMax } from '@/utils/calculations';
 import { formatUnitsToNumber } from '@/utils/numberFormatters';
 
-import { useBeforeBridgeFunds } from '../../Create/SetupEoaFunding/useBeforeBridgeFunds';
 import { TokenRequirement } from '../components/TokensRequirements';
+import { useBeforeBridgeFunds } from './useBeforeBridgeFunds';
```

Comment:
can we please add a TODO to rename this hook? It was originally being used in bridge component hence I didn't change it, but now that this is the only place we are using it, we can change it to maybe `useCreateDummyService`?